src/plugins/login/utils/simpleFileTokenCache.test.ts

```diff
@@ -1,9 +1,12 @@
-import fs from "fs";
 import path from "path";
 import os from "os";
 import mockFs from "mock-fs";
 import { MockFactory } from "../../../test/mockFactory";
 import { SimpleFileTokenCache } from "./simpleFileTokenCache";
+let fs;
+jest.isolateModules(() => {
```

Comment:
What does the `isolateModules()` do?

Reply:
Is it needed since you figured out the mock/restore stuff?

Reply:
Nope. Good catch. I'll remove